omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+Administrators with restricted privileges
+=========================================
+
+
+Summary
+-------
+
+OMERO allows you to create administrators with a subset of the full
+administrator privileges. This is a way to cater for the need for
+more powerful users acting on behalf of all other OMERO users,
+with no group allegiance but with access to all groups and data of
+all users in OMERO. This should be achieved without creating new
+full administrators in OMERO.
+In the real world, these administrators with restricted privileges
+(restricted admins) will typically be imaging
+facility managers, image analysts, or anybody who needs to organize
+users and data of others' in OMERO.
+
+Full administrators in OMERO can create new administrators with
+restricted privileges using the OMERO.web
+interface, see :help:`create new users <sharing-data#admin>` chapter
+of our Help documentation.
+
+We suggest here four setups that should cover the four mainstream
+workflows. Nevertheless, you can combine the privileges
+(check the checkboxes in the OMERO.web interface)
+in any way you see fit. The privilege were designed in such a way
+that they still bear useful functionality even when used in
+isolation. For example checking the :term:`Chown` checkbox will
+indeed give the new administrator with restricted
+privileges the power to transfer ownership of other users'
+data with no other ifs and buts.
```

Comment:
I would remove the `indeed` an ` with no other ifs and buts` here--it doesn't add any extra information.